userguide/README.md

```diff
@@ -59,7 +58,7 @@ An AWS administrator can create application users by following these steps:
 
 1. On the **dashboards** page, under **Edit** mode, you can modify the dashboard.
     ![dashboard-edit-screen.png](./imgs/dashboard-edit-screen.png)
-1. You can update your dashboard with different combinations of the following:
+1. ou can update your dashboard with different combinations of the following:
```

Comment:
Sorry to be so nit-picky, but you lost the 'Y'!

Reply:
updated with the Y added back